packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -6,10 +6,12 @@ import { CSSTransition } from 'react-transition-group';
 import { mobileOSDetect, getPosition } from '@deriv/shared';
 import { listPropType, findNextFocusableNode, findPreviousFocusableNode } from './dropdown';
 import Items from './items.jsx';
-import NativeSelect from './native-select.jsx';
 import DisplayText from './display-text.jsx';
+import SelectNative from '../select-native';
```

Comment:
To avoid duplicated stylesheet imports:

```suggestion
import SelectNative from '../select-native.jsx';
```